### ISSUE TITLE
Logger skips numbers that are surrounded by terminal escape codes

### ISSUE THREAD

**Issue by jalaj-k** [codegods/blogit#21]
The following code...

```python
from logging import getLogger
from helpers.formatter import init
init()
logger = getLogger()
logger.info("Error code: \x1b[31m{}\x1b[0m".format(1))
```

... logs `Error code: 1` on the console which is fine but it logs just `Error code: ` in the file.

**Comment by jalaj-k** [comment]
I am quite sure it's got something to do with the regular expressions that we use to remove terminal escape sequences from log files.

https://github.com/codegods/blogit/blob/18750d8b5ac71b86c012720535344ea2b6e76f5e/helpers/formatter.py#L64-L67

**Comment by jalaj-k** [comment]
This has been resolved in 612a1c1.
